addons/crm_livechat/static/src/core/web/livechat_channel_info_list_patch.xml

```diff
@@ -2,21 +2,11 @@
 <templates xml:space="preserve">
     <t t-name="website_livechat.LivechatChannelInfoList" t-inherit="im_livechat.LivechatChannelInfoList" t-inherit-mode="extension">
         <xpath expr="//t[@t-name='extra_infos']" position="inside">
-            <div t-if="props.thread.livechatVisitorMember?.persona?.opportunity_ids.length" class="d-flex flex-column bg-inherit gap-1">
-                <h3 class="pt-3">Open leads</h3>
-                <t t-foreach="props.thread.livechatVisitorMember?.persona?.opportunity_ids" t-as="lead" t-key="lead.localId">
-                    <a
-                        class="btn btn-sm btn-secondary d-flex align-items-center justify-content-center gap-1 rounded-3"
-                        t-att-href="lead.href"
-                        t-att-data-oe-id="lead.id"
-                        t-att-data-oe-model="'crm.lead'"
-                        contenteditable="false"
-                    >
-                        <i class="fa fa-external-link opacity-75"/>
-                        <span class="fw-bold" t-esc="lead.name"/>
-                    </a>
-                </t>
-            </div>
+            <t t-call="im_livechat.LivechatChannelInfoList.info_links">
+                <t t-set="title" t-value="'Open leads'"/>
+                <t t-set="info_records" t-value="props.thread.livechatVisitorMember?.persona?.opportunity_ids"/>
+                <t t-set="model" t-value="'crm.lead'"/>
+            </t>
```

Comment:
Nitpick: would `t-set` once a `templateParams`.
That way the template params are all in `templateParams`, and the namespace ensures low chance of conflict with context of component template.

Reply:
The previous code was copy-paste, that's not good for code maintenance.
Here this is better but when using `t-call` with variables there's a problem of sharing `ctx` of component context, hence why I suggest to pass `templateParams` as a single param.

I also used `t-call` recently and observed this problem. This sucks but that's still a better tradeoff than what we had before.